Formula/sha1dc.rb

```diff
@@ -0,0 +1,27 @@
+class Sha1dc < Formula
+  desc "Tool to detect SHA-1 collisions in files, including SHAttered"
+  homepage "https://github.com/cr-marcstevens/sha1collisiondetection"
+  url "https://github.com/cr-marcstevens/sha1collisiondetection.git",
+      :revision => "cb10f50d20a1c6c9efbf526c5da510c492d595b9"
+  version "HEAD-cb10f0"
+  head "https://github.com/cr-marcstevens/sha1collisiondetection.git"
+
+  depends_on "libtool" => :build
+
+  def install
+    inreplace "Makefile", "libtool", "glibtool"
+
+    # By default tries to build with HAVEAVX=1, fails.
+    inreplace "Makefile", "HAVEAVX=1", "HAVEAVX=0"
+    system "make"
+
+    bin.install "bin/sha1dcsum"
+    bin.install "bin/sha1dcsum_partialcoll"
+    lib.install "bin/.libs/libdetectcoll.a"
+  end
+
+  test do
+    # Build system's 'make' ran 'make test'
+    system "#{bin}/sha1dcsum"
```

Comment:
Gotcha. While it seems clear they're add a release tag (they have said so), what's the actual homebrew rule here? I suppose I wouldn't have put any of this packaging effort in if I'd realized it would block on the upstream designating a tag (although I did learn more about homebrew in the process...).

It can't possibly be the rule that homebrew won't package software whose upstream doesn't bother to maintain adequate versioning, since that's a not-infrequent issue in reality. Or is the requirement that I would have to fork it myself and publish a tag? (Not a business I want to be in, at least in this particular case, which has active maintenance).

Or is this a maintainer judgement call, i.e. that this package isn't important enough to merit such treatment, but some other package might be? I don't disagree with that evaluation, but I'm just trying to understand the rules of the road.